### PR TITLE
Current Certificate is outdated

### PR DESCRIPTION
**Pull request by sandy2008**

### CODE REVIEW

**Comment by Joel-hanson** [comment]
Thanks for the PR @sandy2008 will merge this as soon I test it.

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 FROM ubuntu:18.04
 
+RUN sudo apt install ca-certificates
```
Comment (Joel-hanson) [inline Dockerfile:3]: Sudo is not required

**Review by Joel-hanson — CHANGES_REQUESTED** [review]
Requested changes

**Review by sandy2008 — COMMENTED** [review]
Thx

**Pull request merged**